kncaputo/fitlit-refactor-tractor
# Form inputs

## What's this PR do?
- Add form buttons and form input so users can add their own data
- Adds a calendar feature so users can chose which day to look at their data

## Where should the reviewer start?
- `index.html` and `styles.scss`

## How should this be manually tested?
- Run `npm start` in the terminal and ensure the buttons and calendar appear
- Unhide the form inputs in `index.html` to see the forms

## Any background context you want to provide?
- These buttons currently have no functionality 

## Screenshots/GIFs
Overall Page: 
<img width="1203" alt="Screen Shot 2020-10-30 at 5 06 05 PM" src="https://user-images.githubusercontent.com/58453566/97764058-08068a00-1ad3-11eb-8ac0-e9a8ea6ae7a6.png">

Form Inputs (will show up separately on button click)
<img width="292" alt="Screen Shot 2020-10-30 at 5 13 09 PM" src="https://user-images.githubusercontent.com/58453566/97764129-3b491900-1ad3-11eb-91e3-7d776c64102b.png">
